developer_manual/basics/_available_events_ocp.rst

```diff
@@ -651,10 +727,7 @@ that it is not possible to use the `disallowXYZ` functions.
 
 .. versionadded:: 18
 
-Event to request a secure password to be generated.
```

Comment:
Should be added to the code but for now could you unstage this change? It removes useful information